src/locales/cy/pages.yml

```diff
@@ -1,5 +1,26 @@
-intro:
-  h1: Intro
+answer-security-questions:
+  title: You need to answer some security questions
```

Comment:
Page title should be translated too

Reply:
I wasn't sure about that - there is no translation in the file: https://docs.google.com/document/d/17clnZncmGrgs2LT7A4O7e1EG7KPTSRmKCsYdTe5_lo0/edit#heading=h.t1dlal7t1ftb
Who should I ask for the translation?

Reply:
(awaiting confirmation from UCD on this)

Reply:
Had confirmation from UCD that the page title and heading should be the same (both translated). So we only need one entry here (suggest keeping `title` and using it for both page title and h1 heading as done in #82)